src/distilabel/llms/huggingface/inference_endpoints.py

```diff
@@ -59,10 +59,35 @@ class InferenceEndpointsLLM(AsyncLLM):
         use_openai_client: whether to use the OpenAI client instead of the Hugging Face client.
 
     Examples:
-        >>> from distilabel.llms.huggingface import AsyncInferenceEndpointsLLM
-        >>> llm = AsyncInferenceEndpointsLLM(model_id="model-id")
-        >>> llm.load()
-        >>> output = await llm.agenerate([{"role": "user", "content": "Hello world!"}])
+        from distilabel.llms.huggingface import InferenceEndpointsLLM
```

Comment:
we need to either put this between a markdown code block or to use `>>>` again to get it render in the UI

![image](https://github.com/argilla-io/distilabel/assets/29572918/34927b04-7f92-48ca-98e7-29824bb0955f)

Reply:
As well as adding the code-comments as `>>> # ...` as @gabrielmbmb mentioned above

Reply:
I went with fences because they support the copy button.